operator/install/install.yaml

```diff
@@ -0,0 +1,6167 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    app: apicurio-registry-operator
+    app.kubernetes.io/component: operator
+    app.kubernetes.io/name: apicurio-registry-operator
+    app.kubernetes.io/part-of: apicurio-registry
+    app.kubernetes.io/version: 3.0.4-SNAPSHOT
+  name: apicurioregistries3.registry.apicur.io
+spec:
+  group: registry.apicur.io
+  names:
+    kind: ApicurioRegistry3
+    plural: apicurioregistries3
+    shortNames:
+    - registry3
+    singular: apicurioregistry3
+  scope: Namespaced
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        properties:
+          spec:
```

Comment:
not needed now, but we need to review the "public API" before we release the first version of this.

Reply:
Agreed.